avogadro/CMakeLists.txt

```
@@ -12,6 +12,8 @@ if(USE_VTK)
   add_definitions(-DAVO_USE_VTK)
 endif()
 
+find_package(JKQtPlotter${QT_VERSION} REQUIRED)
```

https://github.com/OpenChemistry/avogadrolibs/pull/2220

This seems typo

```suggestion
find_package(JKQTPlotter${QT_VERSION} REQUIRED)
```